release-notes/PTP/PTP_1.1.3_CU2.md

```diff
@@ -0,0 +1,79 @@
+---
+uid: PTP_1.1.3_CU2
+---
+
+# PTP 1.1.3 CU2
+
+## Enhancements
+
+#### New parameters added to standard PTP mediation layer [ID_34993]
+
+Parameters for the following connectors that were still missing in the mediation layer have now been added:
+
+- Cisco Nexus
+- Arista Manager
+- Tektronik SPG8000
+
+#### New PTP parameters in Tektronik SPG8000 connector [ID_34994]
+
+The following parameters have been added to the Tektronik SPG8000 connector:
+
+- Clock class (in the PTP Status table)
+- Local Offset time (in the PTP Status table)
+- Step (in the PTP Status table)
+- Local Time (to be added to the DVE and mediated)
+
+#### Improved support for Tektronix SPG8000 [ID_35055]
+
+The following parameters from the Tektronix SPG8000 connector have been added to the mediation layer:
```

Comment:
can you mention name of child connector 'Tektronix SPG8000 - PTP Interface' instead of parent connector 'Tektronix SPG8000'

Reply:
Updated! I noticed that this was also changed for RN 34994, so I've adjusted that one as well.